c/pthread-lit/fkp2014_true-unreach-call.i

```diff
@@ -654,25 +654,26 @@ void __VERIFIER_assert(int cond) {
   }
   return;
 }
-void __VERIFIER_atomic_assert(int cond) {
```

Comment:
`__VERIFIER_atomic_assert ` didn't make sense to me...  The condition gets passed as an int by value, so the value that gets passed into `__VERIFIER_atomic_assert` will be whatever the value of `cond` is at the time of its evaluation, which happens prior to the call to `__VERIFIER_atomic_assert`.  Once `__VERIFIER_atomic_assert` is entered, all variables are local, so the execution order across threads shouldn't matter.

As a result, I decided to get rid of it, and call the standard `__VERIFIER_assert` in its place.

Am I missing something here?